.github/CODEOWNERS

```diff
@@ -20,4 +20,5 @@
  
 
 # for all files:
-* @kumarp20 @bpjain2004 @francis-nijay @harishp8889 @spriya-m @prablr79 @hoppea2 @tdawe @coulof @dattaarindam
+* @hoppea2 @dattaarindam @bpjain2004 @francis-nijay @harishp8889 @kumarp20 @prablr79 @spriya-m @tdawe @coulof
```

Comment:
@harishp8889 , could you please correct this?

Reply:
@spriya-m I have corrected the error, The issue is `dattaarindam` this user is not added as part of this repo and hence still the error is thrown. @prablr79 Can you please add him to this repo